dbms/src/Storages/Page/V3/BlobStore.cpp

```diff
@@ -236,6 +231,8 @@ PageEntriesEdit BlobStore::write(DB::WriteBatch & wb, const WriteLimiterPtr & wr
 
     const size_t all_page_data_size = wb.getTotalDataSize();
 
+    // If the WriteBatch is too big, we will split the Writes in the WriteBatch to different `BlobFile`.
+    // This can avoid allocating a big buffer for writing data and can smooth memory usage.
     if (all_page_data_size > config.file_limit_size)
```

Comment:
Should we handle the case that `write_size + existing_size > config.file_limit_size` ?

Reply:
This is the function which write data to BlobStore. I think we don't need to consider the `existing_size` of BlobStore here.